www/js/shortcut_tray.json

```diff
@@ -0,0 +1,26 @@
+{
+    "anandSahib": {
+        "label": "Anand Sahib",
+        "labelType": "roman",
+        "type": "shabad",
+        "ref": 333375
```

Comment:
i'm guessing this will be updated before the release.

Reply:
Yeah, this will be changed to`type:bani`, and use banidb to load anand sahib + salok. 🙏